src/questions/chapter5.yaml

```diff
@@ -0,0 +1,1075 @@
+    - D
+  topic: Library
+  subTopics:
+    - library
+    - properties
+  difficultyLevel: Low
+  choices:
+    - content: It does not have state variables.
+      key: A
+      order: 0
+    - content: It can not inherit any element.
+      key: B
+      order: 1
+    - content: It can not be inherited
+      key: C
+      order: 2
+    - content: None
+      key: D
+      order: 3
+
+- uuid: 42152571-7564-440b-83c5-79cb91a6c4a7
+  type: SingleChoice
+  content: Choose the correct description about the popular solidity library OpenZeppelin.
+  hint: noHint
+  explanation: OpenZeppelin Contracts helps you minimize risk by using battle-tested libraries of smart contracts for Ethereum and other blockchains. 
+  answerKeys:
+    - C
+  topic: Library
+  subTopics:
+    - library
+    - example
+  difficultyLevel: Low
+  choices:
+    - content: Includes many modular libraries that are very useful for implementation like ArrayUtils, Token, CrowdSale, Vesting, StringUtils, LinkedList, Wallet, etc.
+      key: A
+      order: 0
+    - content: Includes interesting and useful libraries like DoublyLinkedList, StringUtils, IterableMapping, etc.
+      key: B
+      order: 1
+    - content: Includes other supporting libraries are Roles, MerkleProof, ECDSA, Math, Address, SafeERC20, ERC165Checker, SafeMath, Arrays, etc which protects from overflow.
+      key: C
+      order: 2
+    - content: None
+      key: D
+      order: 3
+
+- uuid: 40d58c0b-7475-41f7-abf8-c9af2646f78b
+  type: SingleChoice
+  content: Choose the data type that can not be used in solidity library.
+  hint: noHint
+  explanation: Library in solidity can implement some data types like struct and enums which are user-defined, and constant variables that are stored in a stack of Ethereum, not in storage. 
+  answerKeys:
+    - D
+  topic: Library
+  subTopics:
+    - variables
+  difficultyLevel: Low
+  choices:
+    - content: struct
+      key: A
+      order: 0
+    - content: enums
+      key: B
+      order: 1
+    - content: constant variables
+      key: C
+      order: 2
+    - content: None
+      key: D
+      order: 3
+
+- uuid: 46bab6f7-0612-4b6d-9b09-3e3d8f4db9c4
+  type: SingleChoice
+  content: Which of the following can not be declared inside a library in solidity?
+  hint: noHint
+  explanation: Libraries do not have any storage thus it cannot hold state variables, fallback or payable functions also cannot be created inside the library as it cannot store ethers.
+  answerKeys:
+    - D
+  topic: Library
+  subTopics:
+    - variables
+    - storage
+  difficultyLevel: Low
+  choices:
+    - content: state variable
+      key: A
+      order: 0
+    - content: fallback
+      key: B
+      order: 1
+    - content: None
+      key: C
+      order: 2
+    - content: Both A and B
+      key: D
+      order: 3
+
+- uuid: d608a41e-befd-4633-b74d-aef833e1f8e6
+  type: SingleChoice
+  content: Type of functions those can be called directily from outside of the library.
```

Comment:
`Type of functions those can be called directily from outside of the library.`.  grammatical correction needed. Also spelling of `directily`